modules/configuration-of-stream-labels-in-loki-operator.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * configuring/loki-query-performance-troubleshooting.adoc
+
+:_newdoc-version: 2.18.4
+:_template-generated: 2025-09-25
+:_mod-docs-content-type: CONCEPT
+
+[id="configuration-of-stream-labels-in-loki-operator_{context}"]
+= Configuration of stream labels in {loki-op}
+
+Configuring which labels the {loki-op} will use as stream labels depends on the data model you are using: ViaQ or OpenTelemetry Protocol (OTLP). 
+
+Both models come with a predefined set of stream labels, for more information, see link:https://docs.redhat.com/en/documentation/red_hat_openshift_logging/latest/html/configuring_logging/opentelemetry-data-model[OpenTelemetry data model].
+
+ViaQ model::
+ViaQ does not support structured metadata.
+To configure stream labels for the ViaQ model, add the configuration in the `ClusterLogForwarder` resource. For example:
++
+[source,yaml]
+----
+apiVersion: observability.openshift.io/v1
+kind: ClusterLogForwarder
+metadata:
+  name: instance
+  namespace: openshift-logging
+spec:
+  serviceAccount:
+    name: logging-collector
+  outputs:
+  - name: lokistack-out
+    type: lokiStack
+    lokiStack:
+      target:
+        name: logging-loki 
+        namespace: openshift-logging
+      labelKeys:
+         LokiStackTenantLabelKeys:
```

Comment:
There is no such attribute as LokiStackTenantLabelKeys 
```
    outputs:
    - name: lokistack
      type: lokiStack
      lokiStack:
        authentication:
          token:
            from: serviceAccount
        labelKeys:
          application:
            ignoreGlobal: <true_or_false>
            labelKeys: <array_of_keys>
          audit:
            ignoreGlobal: <true_or_false>
            labelKeys: <array_of_keys>
          infrastructure:
            ignoreGlobal: <true_or_false>
            labelKeys: <array_of_keys>
```

Reply:
fixed